src/Algebra/Graph/AdjacencyIntMap/Algorithm.hs

```diff
@@ -35,8 +35,12 @@ import qualified Data.Graph.Typed   as Typed
 import qualified Data.IntMap.Strict as IntMap
 import qualified Data.IntSet        as IntSet
 
--- | Compute the forest of a graph's vertices in breadth first order. Complexity:
--- /O(n+m*log n)/ time and /O(n+m)/ space.
+-- | Compute the /breadth-first search/ forest of a graph, where
+--   adjacent vertices are expanded smallest to biggest with respect
```

Comment:
I think "are expanded smallest to biggest" could be phrased a bit more precisely as "are explored in the increasing order".